docs/modules/ROOT/partials/nav-howtos.adoc

```diff
@@ -1,7 +1,15 @@
-* xref:app-catalog:ROOT:how-tos/implement_new_service_offering.adoc[Implement New Service Offering]
-
 * Crossplane
-** xref:app-catalog:ROOT:how-tos/crossplane/investigate_service_instances.adoc[Investigate Service]
+** xref:app-catalog:ROOT:how-tos/crossplane/investigate_service_instances.adoc[Investigate a Service]
+** xref:app-catalog:ROOT:how-tos/crossplane/implement_new_service_offering.adoc[Implement a New Service]
+
+* Crossplane Service Broker
+** xref:app-catalog:ROOT:how-tos/crossplane_service_broker/overview.adoc[Overview]
+** xref:app-catalog:ROOT:how-tos/crossplane_service_broker/setup_crossplane_service_broker.adoc[Setup a _Crossplane Service Broker_]
+** xref:app-catalog:ROOT:how-tos/crossplane_service_broker/setup_service_catalog.adoc[Setup a _Service Catalog_]
+** xref:app-catalog:ROOT:how-tos/crossplane_service_broker/bearer_token_authentication.adoc[HTTP _Bearer Token_ authentication]
+** xref:app-catalog:ROOT:how-tos/crossplane_service_broker/connect_service_catalog_to_service_broker.adoc[Connect the _Service Catalog_ to the _Service Broker_]
+** xref:app-catalog:ROOT:how-tos/crossplane_service_broker/basic_authentication.adoc[HTTP _Basic_ authentication]
+** xref:app-catalog:ROOT:how-tos/crossplane/implement_new_service_offering.adoc[Implement a New Service]
```

Comment:
This page is now twice linked in the nav?

Reply:
I couldn't decide where it'd fit more, so I added it to both sections. I've now removed it from the _Crossplane_ section, since I beleive it's more tailored towards the _Crossplane Service Broker_ than _Crossplane_ itself.